.github/workflows/.deploy-main.yaml

```diff
@@ -9,7 +9,7 @@ on:
         type: string
 
 env:
-  image: ghcr.io/${{ github.repository }}/etterlatte-${{ inputs.applikasjon }}:main
+  image: europe-north1-docker.pkg.dev/nais-management-233d/etterlatte/${{ github.repository }}-etterlatte-${{ inputs.applikasjon }}:main
```

Comment:
Ref også https://nav-it.slack.com/archives/C5KUST8N6/p1680248252666829?thread_ts=1680178447.627559&cid=C5KUST8N6